components/Dashboard.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div>
```

Comment:
Honestly, at this point I'm not even convinced this `header` element justifies its own component and we could simply have this markup directly in the `<template>` for `index.vue`, but I'm fine keeping it as a separate component if you have some bigger vision for what this might eventually become.